desktop_version/src/titlerender.cpp

```diff
@@ -279,106 +279,64 @@ void titlerender(Graphics& dwgfx, mapclass& map, Game& game, entityclass& obj, U
-            dwgfx.Print( 80, 60,"Ido Yehieli", tr, tg, tb);
-            dwgfx.Print( 80, 70,"Jade Vault Games", tr, tg, tb);
-            dwgfx.Print( 80, 80,"James Andrews", tr, tg, tb);
-            dwgfx.Print( 80, 90,"James Riley", tr, tg, tb);
-            dwgfx.Print( 80, 100,"James Hsieh", tr, tg, tb);
-            dwgfx.Print( 80, 110,"Jasper Byrne", tr, tg, tb);
-            dwgfx.Print( 80, 120,"Jedediah Baker", tr, tg, tb);
-            dwgfx.Print( 80, 130,"Jens Bergensten", tr, tg, tb);
-            dwgfx.Print( 80, 140,"Jeremy J. Penner", tr, tg, tb);
-            dwgfx.Print( 80, 150,"Jeremy Peterson", tr, tg, tb);
-            dwgfx.Print( 80, 160,"Jim McGinley", tr, tg, tb);
-            dwgfx.Print( 80, 170,"Jonathan Cartwright", tr, tg, tb);
-        }
-        else if (game.currentmenuname == "credits7")
-        {
-            dwgfx.Print( -1, 20, "and also by", tr, tg, tb, true);
+            int maxheight = 10 * 14;
+            int totalheight = (endidx - startidx) * 10;
+            int emptyspace = maxheight - totalheight;
 
-            dwgfx.Print( 80, 40,"John Nesky", tr, tg, tb);
-            dwgfx.Print( 80, 50,"Jos Yule", tr, tg, tb);
-            dwgfx.Print( 80, 60,"Jose Flores", tr, tg, tb);
-            dwgfx.Print( 80, 70,"Josh Bizeau", tr, tg, tb);
-            dwgfx.Print( 80, 80,"Joshua Buergel", tr, tg, tb);
-            dwgfx.Print( 80, 90,"Joshua Hochner", tr, tg, tb);
-            dwgfx.Print( 80, 100,"Kurt Ostfeld", tr, tg, tb);
-            dwgfx.Print( 80, 110,"Magnus P~lsson", tr, tg, tb);
-            dwgfx.Print( 80, 120,"Mark Neschadimenko", tr, tg, tb);
-            dwgfx.Print( 80, 130,"Matt Antonellis", tr, tg, tb);
-            dwgfx.Print( 80, 140,"Matthew Reppert", tr, tg, tb);
-            dwgfx.Print( 80, 150,"Michael Falkensteiner", tr, tg, tb);
-            dwgfx.Print( 80, 160,"Michael Vendittelli", tr, tg, tb);
-            dwgfx.Print( 80, 170,"Mike Kasprzak", tr, tg, tb);
-        }
-        else if (game.currentmenuname == "credits8")
-        {
-            dwgfx.Print( -1, 20, "and also by", tr, tg, tb, true);
+            int yofs = 40 + (emptyspace / 2);
 
-            dwgfx.Print( 80, 70,"Mitchel Stein", tr, tg, tb);
-            dwgfx.Print( 80, 80,"Sean Murray", tr, tg, tb);
-            dwgfx.Print( 80, 90,"Simon Michael", tr, tg, tb);
-            dwgfx.Print( 80, 100,"Simon Schmid", tr, tg, tb);
-            dwgfx.Print( 80, 110,"Stephen Maxwell", tr, tg, tb);
-            dwgfx.Print( 80, 120,"Swing Swing Submarine", tr, tg, tb);
-            dwgfx.Print( 80, 130,"Tam Toucan", tr, tg, tb);
-            dwgfx.Print( 80, 140,"Terry Dooher", tr, tg, tb);
-            dwgfx.Print( 80, 150,"Tim W.", tr, tg, tb);
-            dwgfx.Print( 80, 160,"Timothy Bragan", tr, tg, tb);
-        }
-        else if (game.currentmenuname == "credits9")
+            for (int i = startidx; i < startidx + 14 && i < game.patrons.size(); ++i)
```

Comment:
Forgot an `(int)` here, compiler spits out a warning on this line.

Reply:
The whole loop condition can actually be changed to `i < endidx`

Reply:
Done

Reply:
Looks good, thanks!